src/ScottPlot5/ScottPlot5/Fonts.cs

```diff
@@ -5,6 +5,37 @@
 /// </summary>
 public static class Fonts
 {
+    /* Typefaces are cached to improve performance.
+     * https://github.com/ScottPlot/ScottPlot/issues/2833
+     * https://github.com/ScottPlot/ScottPlot/pull/2848
+     */
+    private static readonly Dictionary<int, SKTypeface> TypefaceCache = new();
```

Comment:
I'm not sure why you're going to the trouble of making this a `Dictionary<int, SKTypeface>` and all the associated hash combining, I'm pretty sure you can just use a tuple like `(string fontname, bool isBold, bool isItalic)` as a dictionary key. This saves you work, and it will be robust to hash collisions.

Reply:
I agree with your suggestion to simplify the code.